arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md

```diff
@@ -1,17 +1,67 @@
 ---
-title: 'Introduction to the Arbitrum Orbit SDK'
-sidebar_label: 'Orbit SDK: Introduction'
-description: 'introduction to the Arbitrum Orbit SDK'
-author: Mehdi Salehi
-sme: Mehdi Salehi
-target_audience: 'Developers deploying and maintaining Orbit chains.'
-sidebar_position: 0
+title: 'Get started with the Arbitrum Orbit SDK'
+sidebar_label: 'Orbit SDK: Get started'
+description: 'Learn how to deploy and manage your Orbit chain with the Arbitrum Orbit SDK.'
+author: GreatSoshiant
+sme: GreatSoshiant
+user_story: As a current or prospective Orbit chain owner, I need to onboard into the Orbit SDK by understanding the available onboarding paths, and how to select the path that meets my needs.
+content_type: get-started
 ---
 
 import PublicPreviewBannerPartial from './partials/_orbit-public-preview-banner-partial.md';
 
 <PublicPreviewBannerPartial />
 
+The Arbitrum Orbit SDK lets you programmatically create and manage your own Orbit chain(s). Its capabilities include:
```

Comment:
I've made an assumption: The purpose of this document is to onboard readers into the Orbit SDK as efficiently as possible. To that end, I've attempted to:

 - remove extraneous copy
 - optimize for actionable steps, decision-making, and skimmability

Reply:
I like your copy better than the original.